build_dbg_j2.sh

```diff
@@ -0,0 +1,45 @@
+#! /bin/bash
+
+echo "
+FROM $1
+
+ARG docker_container_name
+
+## Make apt-get non-interactive
+ENV DEBIAN_FRONTEND=noninteractive
+
+{% if $2 is defined %}
+{% if $2|length %}
+
+COPY \
+{% for deb in $2.split(' ') -%}
+debs/{{ deb }}{{' '}}
+{%- endfor -%}
+debs/
+
+RUN dpkg -i \
+{% for deb in $2.split(' ') -%}
+debs/{{ deb }}{{' '}}
+{%- endfor %}
+
+{% endif %}
+{% endif %}
+
+{% if $3 is defined %}
+{% if $3|length %}
+
+RUN apt-get install -f -y \
+{% for dbg in $3.split(' ') -%}
+{{ dbg }}{{' '}}
+{%- endfor %}
+
+{% endif %}
+{% endif %}
+
+
+## Clean up
+RUN apt-get clean -y; apt-get autoclean -y; apt-get autoremove -y
+RUN rm -rf /debs
+
+CMD ["/usr/bin/supervisord"]
```

Comment:
this should be the same as the base docker, it should be removed.

Reply:
Actually, it loses it significance as base docker has ENTRYPOINT. 

BTW, I would like to make it CMD in the base docker image, which allows an easy override. But it can wait, until a bigger change is in demand.

Reply:
done